ReplayBrowser/Pages/Shared/SearchBar.razor

```diff
@@ -36,15 +44,24 @@
                 </button>
             </div>
         </div>
-    </div>
-
-    <div class="d-flex" style="margin-top: 1rem;">
-        <button class="btn btn-outline-primary" style="margin-right: 1rem" type="button" id="addSearchBar">
-            +
-        </button>
-        <button class="btn btn-outline-success" type="button" onclick="search(0)">Search</button>
-    </div>
+    }
+}
+
+<div class="search-bar">
+@{
+    RenderLine(null);
```

Comment:
Could probably leave a comment here for future devs that this is meant to draw the prefab. Was a bit confused at first :p